src/i_midimusic.c

```diff
@@ -740,18 +524,12 @@ static void SendSysExMsg(unsigned int delta_time, const midi_event_t *event)
         }
 
         // Replace SysEx part level message with channel volume message.
-        SendManualVolumeMsg(delta_time, channel, data[7]);
+        SendManualVolumeMsg(channel, data[7]);
         return;
     }
 
     // Send the SysEx message.
-    native_event.dwDeltaTime = delta_time;
-    native_event.dwStreamID = 0;
-    native_event.dwEvent = MAKE_EVT(length + sizeof(byte), 0, 0, MEVT_LONGMSG);
-    WriteBuffer((byte *)&native_event, sizeof(native_event_t));
-    WriteBuffer(&event_type, sizeof(byte));
-    WriteBuffer(data, length);
-    WriteBufferPad();
+    SendLongMsg(data, length);
```

Comment:
`MIDI_EVENT_SYSEX` must be inserted at the front like before otherwise SysEx messages don't work.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/1635/commits/86f7a213b6d2cc5747721d2ba7995e4aacdd2c63